src/commands/e2e/ConfigTemplate.ts

```diff
@@ -0,0 +1,62 @@
+import {E2E} from './E2E';
+
+export class ConfigTemplate {
+    private readonly template: string;
+
+    constructor(e2eFolder: string, browser: string, baseUrl: string, timeout: number) {
```

Comment:
We have to test this under Windows to make sure the generated paths are used correctly 👍

Reply:
I tested it with Audrius on Windows and fixed e2eFolder, should be fine now